client/src/actions/auth.js

```diff
@@ -131,15 +131,14 @@ export const login = (email, password) => async (dispatch) => {
   const body = JSON.stringify({ email, password });
 
   try {
-    // const res = await axios.post('/api/login', body, config);
-    const placeholder = { token: "token-placeholder" };
-    const res = { data: placeholder };
+    const res = await axios.post(`${URL_HOST}/users/login`, body, config);
```

Comment:
Could you please check the back-end and double check if this api route works? I tried couple times at my end, and no luck making it work.